uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -6,7 +6,7 @@
     </button>
     <h4 class="modal-title" ng-switch="isRole('admin')">
       <span ng-switch-when="true">New Document</span>
-      <span ng-switch-when="false">{{getDocumentType(document.activity_type_id)}}</span>
+      <span ng-switch-when="false">{{getDocumentType(document.activity_type_id) || 'Select Document Type'}}</span>
```

Comment:
so the logic here is:

If the modal is being opened by an admin, then *always* show as a title "New Document", otherwise if it's being opened by a staff or a manager, show as a title either the document type or a message asking to select the type

but why does that admin always see "New Document"? What if the admin open an already existing document?